src/TC/third_party/nvEncodeAPI.h

```diff
@@ -0,0 +1,4280 @@
+/*
+ * This copyright notice applies to this header file only:

```

Comment:
vendoring this API has the advantage that we can control the version that works and automatically upgrade users. Disadvantage is that the user can no longer choose the an alternative header and that each header has a minimum driver requirement

Maybe we could add a CMake option to use the interface from the VIDEO_CODEC_SDK if set

Reply:
I allowed to set a custom interface dir via a CMAKE_VARIABLE